pyro/infer/importance.py

```diff
@@ -2,7 +2,6 @@
 import torch
 import warnings
 
-from pyro.distributions.util import logsumexp
```

Comment:
This leads to a wrong display for logsumexp http://docs.pyro.ai/en/0.3.0-release/inference_algos.html#pyro.infer.importance.logsumexp